rpc/core/status.go

```diff
@@ -45,11 +45,12 @@ func Status(ctx *rpctypes.Context) (*ctypes.ResultStatus, error) {
 	result := &ctypes.ResultStatus{
 		NodeInfo: p2pTransport.NodeInfo().(p2p.DefaultNodeInfo),
 		SyncInfo: ctypes.SyncInfo{
-			LatestBlockHash:   latestBlockHash,
-			LatestAppHash:     latestAppHash,
-			LatestBlockHeight: latestHeight,
-			LatestBlockTime:   latestBlockTime,
-			CatchingUp:        consensusReactor.FastSync(),
+			LatestBlockHash:    latestBlockHash,
+			LatestAppHash:      latestAppHash,
+			CurrentBlockHeight: blockStore.Height(),
+			HighestBlockHeight: consensusState.GetLastHeight(),
```

Comment:
Won't HighestBlockHeight be the same as CurrentBlockHeight?

Reply:
Yes, correct @melekes. For now, we can close this PR. Later, I will expose internal info in the reactor and then, I will create separate PR for this.